src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/SourceItemHandler.cs

```diff
@@ -121,6 +121,8 @@ public override Task OnContextReleasedAsync(IWorkspaceProjectContext context)
 
         private void RemoveSourceFile(string fullPath, IWorkspaceProjectContext context)
         {
+            fullPath = _project.MakeRooted(fullPath);
```

Comment:
Seems like we are still building with a very old version MS.VS.LS NuGet package: https://github.com/dotnet/roslyn-project-system/blob/master/src/Dependencies/Roslyn/project.json#L4

Works fine after upgrading to rc2 bits.